packages/knip/src/plugins/postcss/index.ts

```diff
@@ -27,10 +27,19 @@ const resolveConfig: ResolveConfig<PostCSSConfig> = config => {
     : [];
 };
 
+const resolve: Resolve<PostCSSConfig> = async (options, config) => {
+  for (const plugin of await resolveConfig(options, config)) {
+    // Because postcss is not included in peerDependencies of tailwindcss
+    if (plugin === 'tailwindcss') return ['postcss'];
```

Comment:
What's the reason for not adding a check for `tailwindcss` in `config.plugins` to the `resolveConfig` function directly?

Reply:
I misunderstood the implementation policy🙏

FIxed!
https://github.com/webpro-nl/knip/pull/764/commits/03a7583d1f1b96d4a7b1b8675d72b09882874c55

Reply:
Sorry, there was a mistake.

Reply:
Sorry taro. Is there anything in the documentation or code that could have prevented this? Maybe we can improve it for the next contributor.

Thanks for the fix!

Reply:
> Sorry taro. Is there anything in the documentation or code that could have prevented this? Maybe we can improve it for the next contributor.

I re-read the [documentation](https://knip.dev/guides/writing-a-plugin#6-resolveconfig) again, and it succinctly describes how to use `resolveConfig`, so I thought it was fine as it is.

My misunderstanding was due to overcomplicating the issue, as it was a slightly exceptional modification, such as adding the plugin's dependency itself (postcss).